lms/services/vitalsource/_schemas.py

```diff
@@ -35,7 +35,7 @@ class Meta:
 
         @post_load
         def set_url(self, in_data, **_kwargs):
-            # pylint:disable=import-outside-toplevel
```

Comment:
Actual cycle, we were importing in the function to avoid it.

Reply:
There's also some weirdness about this return value. We return `{"table_of_contents": { ... }}`. The only place this is used then is:

```python
    @view_config(route_name="vitalsource_api.books.toc")
    def table_of_contents(self):
        book_id = self._get_book_id()

        book_toc = self.vitalsource_service.book_toc(book_id)
        return book_toc["table_of_contents"]
```

Which looks like it should be:

```python
    @view_config(route_name="vitalsource_api.books.toc")
    def table_of_contents(self):
        return self.vitalsource_service.book_toc(self._get_book_id())
```